website/events/services.py

```diff
@@ -187,8 +187,8 @@ def update_registration(
 
     if (
         not event_permissions(member, event, name)["update_registration"]
-        or not field_values
-    ):
+        and is_organiser(member, event)
```

Comment:
When I am not an organiser this will always allow editing the event fields from the user perspective.

Reply:
The change should probably in `event_permissions`. If you look at this PR: https://github.com/svthalia/concrexit/pull/1121/files, you see that the if statements have not the same meaning anymore. 

First, it checked for either an authenticated member or a name. Now it always requires a member first. That is more likely to be the problem here.

This change does not look like the right solution without causing new problems.

Reply:
Ok I will look it up again on Monday.